### DISCUSSION TITLE
Switching Base Docker Image to UBI

### DISCUSSION THREAD

**Discussion by krishvoor** [strimzi/strimzi-kafka-operator#4489] [Q&A]
The base image is built upon CentOS 7, thus resultant docker images which leverages kafka-base are also on CentOS 7, which is older than UBI 8.  It would be better to equip UBI 8 ( latest and greatest ) and continue the builds, to ensure we are leveraging newer versions/capabilities of dependent packages coming from OS libraries or otherwise.

**Comment by scholzj** [comment]
We are thinking about this. But I do not think it really has newer packages per-se. Especially with Java based applications which anyway rely mostly on JDK and you can have the same JDK version on both.
